gitpushmethod.txt

```diff
@@ -0,0 +1 @@
+tslakjdhgaljkshglasjkhga
```

Comment:
needs an extra semicolon

Reply:
dont need semi colons in javascript @alexjr1194